cli/src/swam/cli/Main.scala

```diff
@@ -32,19 +32,29 @@ object Main extends CommandIOApp(name = "swam-cli", header = "Swam from the comm
   type AsIIO[T] = AsInterface[T, IO]
   type AsIsIO[T] = AsInstance[T, IO]
 
+  ////// CLI-COMMAND ARGUMENTS //////
+  // Beware (Opts.argument != Opts.arguments) && (Opts.option != Opts.options) && (Opts.flag != Opts.flags)
+
   val wasmFile =
     Opts.argument[Path](metavar = "wasm")
 
+  // TODO: What is this for? Change this to Opts.options for more clarity (more verbose as well)?
```

Comment:
These are not options, these are arguments that get passed to the wasm code you execute. They are available through WASI `args_get`.

Reply:
Okay, that makes sense. I was just annoyed that it did not provide any "help" parameter. I would have set it up the same way I set up wasmArgTypes. But then I'll just add a comment.